components/central-application-gateway/README.md

```diff
@@ -27,60 +27,64 @@ To install the Central Application Gateway, follow these steps:
+- **applicationSecretsNamespace** is the Namespace in which the Application secrets used by Central Application Gateway exist. The default Namespace is `kyma-integration`.
+- **requestTimeout** is the timeout for requests sent through Central Application Gateway, expressed in seconds. The default value is `1`.
 - **requestLogging** is the flag for logging incoming requests. The default value is `false`.
 - **proxyTimeout** is the timeout for requests sent through the proxy, expressed in seconds. The default value is `10`.
 - **proxyCacheTTL** is the time to live of the remote API information stored in the proxy cache, expressed in seconds. The default value is `120`.
 
 
 ## API
-The Central Application Gateway exposes:
+
+Central Application Gateway exposes:
 - an external API implementing a health endpoint for liveness and readiness probes
 - 2 internal APIs implementing a proxy handler accessible via a service of type `ClusterIP`
 
-### Standalone (legacy) mode
+### Standalone mode
+
 The proxy API exposes the following endpoint:
 ```bash
 {APPLICATION_NAME}/{SERVICE_NAME}/{TARGET_API_PATH}
 ``` 
 
-For instance, if the user registered the `cc-occ-commerce-webservices` service in the `ec` application using Application Registry, they can send a request to the following URL: 
+For instance, if there's a `cc-occ-commerce-webservices` service in the `ec` Application CR, the user can send a request to the following URL: 
 ```bash
 http://central-application-gateway.kyma-system:8080/ec/cc-occ-commerce-webservices/basesites
 ```
 
-As a result, the Central Application Gateway:
-1. Looks for the `cc-occ-commerce-webservices` service in the `ec` Application CRD and extracts the target URL path along with the authentication configuration
-2. Modifies the request to include the authentication data
+As a result, Central Application Gateway:
+1. Looks for the `cc-occ-commerce-webservices` service in the `ec` Application CR and extracts the target URL path along with the authentication configuration.
+2. Modifies the request to include the authentication data.
 3. Sends the request to the following path:
    ```bash
-   {TARGET_URL_EXTRACTED_FROM_APPLICATION_CRD}/basesites
+   {TARGET_URL_EXTRACTED_FROM_APPLICATION_CR}/basesites
+   ```
 
 #### Standalone mode for Compass - simplified API
 
 The standalone mode can also be used for Compass bundles with a single API definition.
-This means that `{API_DEFINITION_NAME}` should be removed from the URL and its pattern looks like:
-```
+In this case, `{API_DEFINITION_NAME}` is removed from the URL and the pattern looks as follows:
+```bash
 {APPLICATION_NAME}/{API_BUNDLE_NAME}/{TARGET_API_PATH}
 ```
-> **NOTE:** Invocation of service bundles configured with multiple API definitions will result in **400 Bad Request** failure.
+> **NOTE:** Invocation of service bundles configured with multiple API definitions will result in a `400 Bad Request` failure.
```

Comment:
```suggestion
> **NOTE:** Invocation of service bundles configured with multiple API definitions results in the `400 Bad Request` failure.
```

Reply:
Actually, I'd leave `a` here, because the underlying cause for this error can be different things, so in reality it's rather an error of the `400 Bad Request` sort.

Reply:
I'll edit your comment and apply the first part (`will result` > `results`).